components/list/ListItem.js

```diff
@@ -80,7 +86,14 @@ const factory = (ripple, ListItemLayout, ListItemContent) => {
       const children = this.groupChildren();
       const content = <ListItemLayout theme={theme} {...children} {...other} />;
       return (
-        <li className={`${theme.listItem} ${className}`} onClick={this.handleClick} onMouseDown={onMouseDown} onTouchStart={onTouchStart}>
+        <li
```

Comment:
this is a start but the list item is pretty broken in our app.  also if the contents has an a tag in there then you need to tab over it twice and only the inner tab will trigger the link move.

are we doing the aria stuff in a different commit?